src/main/scala/strawman/collection/View.scala

```diff
@@ -138,12 +138,12 @@ object View extends IterableFactoryLike[View] {
     /** The view consisting of all elements of the underlying collection
      *  that satisfy `p`.
      */
-    val first = Partitioned(this, true)
+    val first = Partitioned(this, cond = true)
 
     /** The view consisting of all elements of the underlying collection
      *  that do not satisfy `p`.
      */
-    val second = Partitioned(this, false)
+    val second = Partitioned(this, cond = false)
```

Comment:
Just a minor remark: in general I would appreciate if this kind of code style improvements were in separate commits. That would make the review process easier.